march_simulation/scripts/to_world_transform

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python
+import rospy
+import tf2_ros
+from geometry_msgs.msg import TransformStamped
+from gazebo_msgs.srv import GetModelState, GetModelStateRequest
+
+rospy.init_node('to_world_transform', anonymous=True)
+rospy.loginfo("Exoskeleton is mobile")
+
+transform = TransformStamped()
+transform.header.frame_id = 'world'
+transform.child_frame_id = 'base_link'
+tf_broadcaster = tf2_ros.TransformBroadcaster()
+
+rospy.wait_for_service('/gazebo/get_model_state')
+get_model_srv = rospy.ServiceProxy('/gazebo/get_model_state', GetModelState)
+model = GetModelStateRequest()
+model.model_name = 'march'
+
+rate = rospy.Rate(10)
+
+while not rospy.is_shutdown():
+    try:
+        result = get_model_srv(model)
+        transform.header.stamp = rospy.Time.now()
+        transform.transform.translation = result.pose.position
+        transform.transform.rotation = result.pose.orientation
+
+        tf_broadcaster.sendTransform(transform)
+    except rospy.ServiceException as e:
+        rospy.loginfo("Get Model State service call failed:  {0}".format(e))
+    rate.sleep()
```

Comment:
you should `try` this since now it throws an exception when it is sleeping when you close the node.
```suggestion
    try:
        rate.sleep()
    except ROSInterruptException:
        pass
```

Reply:
Why is this not necessary for other nodes?

Reply:
Most nodes use `rospy.spin` which handles this itself. I had this exact same problem in the state machine a while back.